src/components/hero-image/hero-image.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import heroImage from '~/assets/img/heroImage.jpg';
+import styles from './hero-image.module.scss';
+import { StretchableLink } from '../stretchable-link/stretchable-link';
+
+export interface HeroImageProps {
+    subTitle: string;
+    title: string;
+    className?: string;
+    linkLabel?: string;
+    linkCategorySlug?: string;
+}
+
+export const HeroImage = ({
+    subTitle,
+    title,
+    linkLabel,
+    linkCategorySlug,
+    className,
+}: HeroImageProps) => {
+    return (
+        <div className={classNames(styles.root, className)}>
+            <picture>
+                <img src={heroImage} className={styles.image} alt="Hero banner" />
+            </picture>
```

Comment:
There's no point using `picture` if you're not going to have multiple images for different resolutions.

Reply:
Check if you can use image from wix media manager instead of putting this image into their project that they will need to unbind and remove manually.

Reply:
What about an image from media platform?

Reply:
I got it. I can use it. Updated.